client/src/pages/Signup.jsx

```diff
@@ -34,7 +35,7 @@ function Signup({ updateAppProfile }) {
   const [firstName, setFirstName] = useState('');
   const [lastName, setLastName] = useState('');
   const [email, setEmail] = useState('');
-  const [serviceArea, setServiceArea] = useState('AV');
+  const [serviceArea, setServiceArea] = useState(serviceAreas[0]);
```

Comment:
is there a possibility that the serviceAreas array could be empty when the useState is first called?

Reply:
Probably not, because I'm importing the service areas array from constants.js, which will always have some value in the array